### PR TITLE
Edit the text for flow, grammar, and spelling.

### PR DESCRIPTION
**Pull request by nickpascucci**
I cleaned up the text a bit to flow more smoothly and cleaned up some minor spelling and grammar errors along the way.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -23,93 +23,105 @@
...
-      needs, internet and social network interactions, hacking,
-      coding, managing to-do's and organizing your daily schedule,
-      play Tetris and it can even provide you with free psychotherapy and
-      counseling using <code>M-x doctor</code>.
+      You can use Emacs for all of your text processing needs,
+      internet and social network interactions, hacking, coding,
+      managing to-do's and organizing your daily schedule, playing
+      Tetris, and many other awesome things. It can even provide you
+      with free psychotherapy and counseling using the built
+      in <code>doctor</code>. A vibrant community of hackers are
```
Comment (Fuco1) [inline index.html]: Community is singular, so it should use `is` or just `hackers are`.  I'd go with `is`.

File: index.html [hunk 2]
```diff
@@ -23,93 +23,105 @@
...
-      There are tools for probably every programming language out
-      there. Lisp, Ruby, Python, PHP, Java, Erlang, JavaScript, C,
-      C++, Prolog, Tcl, AWK, PostScript, Clojure, Scala,
-      Perl, Haskell, Elixir all of these languages and more are
-      supported in Emacs.
+      There are tools for every programming language out there. Lisp,
+      Ruby, Python, PHP, Java, Erlang, JavaScript, C, C++, Prolog,
+      Tcl, AWK, PostScript, Clojure, Scala, Perl, Haskell, Elixir all
+      of these languages and more are supported in Emacs. Because of
+      the powerful Lisp core, Emacs is easy to extend to new languages
```
Comment (Fuco1) [inline index.html]: easy to extend **for** new languages?  I'm not sure, but it seems slightly better to me.  With `to` it misses a verb... easy to extend to **add support** or something.  What do you think?

File: index.html [hunk 3]
```diff
@@ -118,47 +130,44 @@
     <h2>Installing Emacs</h2>
     <img src="img/emacs-icon.png" alt="Install Emacs" class="right">
     <p>
-      You can install Emacs on almost every operating system out
-      there. The officially supported systems are GNU, GNU/Linux,
-      FreeBSD, NetBSD, OpenBSD, Mac OS X, MS Windows and Solaris.
+      Emacs is easy to install on almost every operating system out
+      there. Officially supported systems include Mac OS X, Microsoft
```
Comment (Fuco1) [inline index.html]: GNU/Linux should come first, for obvious reasons

File: index.html (hunk 2 above)
Comment (nickpascucci) [reply index.html]: Sure, sounds good.

File: index.html (hunk 1 above)
Comment (nickpascucci) [reply index.html]: SGTM.

File: index.html (hunk 3 above)
Comment (nickpascucci) [reply index.html]: Done.

**Pull request merged**